vsintegration/ProjectTemplates/TutorialProject/Template/Tutorial.fsx

```diff
@@ -192,6 +214,78 @@ module Tuples =
     printfn "Struct Tuple: %A\nReference tuple made from the Struct Tuple: %A" sampleStructTuple (sampleStructTuple |> convertFromStructTuple)
 
 
+/// The F# pipe operators ('|>', '<|', etc.) and F# composition operators ('>>', '<<')
+/// are used extensively when processing data.  These operators are themselves functions
+/// which make use of Partial Application.
```

Comment:
Typo in my comment - I meant "need not be capitalized".  Sorry.  It looks somehow wrong, as if Making It Mysterious